cluster-autoscaler/simulator/utilization/info_test.go

```diff
@@ -33,6 +34,19 @@ func TestCalculate(t *testing.T) {
 	testTime := time.Date(2020, time.December, 18, 17, 0, 0, 0, time.UTC)
 	pod := BuildTestPod("p1", 100, 200000)
 	pod2 := BuildTestPod("p2", -1, -1)
+	podWithInitContainers := BuildTestPod("p-init", 100, 200000)
```

Comment:
I'd add a test case for non-restartable initContainers, as they're treated completely differently from restartable ones, and we're changing the calculations to include both kinds.

Reply:
Sure, I've extended the tests.